src/cmdlib.py

```diff
@@ -146,6 +146,16 @@ def rm_allow_noent(path):
         pass
 
 
+
+# See the "original/canonical" version of this in cmdlib.sh
```

Comment:
Minor: could also do like `get_build_dir` and just have the `cmdlib.sh` version call out to python3 so we maintain only a single canonical version. It's a bit hacky, though I think it's worth it for maintaining a single canonical version (and I suspect we'll reuse that pattern more and more as we port other things).

Reply:
OK done...though I can feel the slowness of the Python interpreter booting when I do `cosa run` now but...oh well.